e-commerce/README.md

```diff
@@ -53,8 +53,18 @@ Each of the four parts of the demo must run in its own terminal window.
 For each setup, each package has a single npm command that is used to build and launch the package
 
 * For payment-backend and shop-backend, run `npm run start` to build and launch the app
+
 * For shop-frontend, run `npm run dev` to launch the app
+  * To run with a backend in the cloud
+  * In lib/backend.ts 
+  * change export const backendAddress = "http://localhost:8082";
+  * to export const backendAddress = "url to cloud backend";
```

Comment:
Can we make these bullet points a sentence? Likewise below

Reply:
fixed